.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: scala
+scala: 2.12.7
+
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot/
+    - $HOME/.sbt/launchers
+
+jdk:
+  - oraclejdk8
+
+script:
+  - set +e
+  - src/main/g8/sbt -Dfile.encoding=UTF8 -J-XX:ReservedCodeCacheSize=256M g8
+  - cd target/g8
+  - ../../src/main/g8/sbt test
```

Comment:
can we do `$HOME/src/main/g8/sbt` here instead?

Reply:
Isn't that less resilient though as that is effectively an absolute path?

Reply:
yeah, that's a good point.